modules/administration/pages/auditing.adoc

```diff
@@ -1,108 +1,87 @@
-The system is affected by the vulnerability, {productname} has at least one patch for it in a channel that is directly assigned to the system.
-
-image:patch-status-not-affected.png[scaledwidth=1em] Grey - Not affected:::
-The system does not have any packages installed that are patchable.
-
-image:patch-status-patched.png[scaledwidth=1em] Green - Patched:::
-A patch has already been installed.
-
-In other words, it can mean the following:
-
-* More than one patch might be needed to fix a certain vulnerability.
-* The image:patch-status-affected-patch-applicable.png[scaledwidth=1em] Orange - state is displayed when {productname} has at least one patch in an assigned channel.
-This might mean that, after installing such a patch, others might be needed—users should double check the CVE Audit page after applying a patch to be sure that their systems are not affected anymore.
-
-For a more precise definitions of these states, see <<cve-tips>>.
-
-[NOTE]
-.Unknown CVE Number
+[IMPORTANT]
 ====
-If the CVE number is not known to {productname}, an error message is displayed because {productname} cannot collect and display any audit data.
+You must apply CVEs to your clients as soon as they become available.
 ====
 
-For each system, the [guimenu]``Next Action`` column contains suggestions on the steps to take to address the vulnerabilities.
-Under these circumstances it is either sensible to install a certain patch or assign a new channel.
-If applicable, a list of "`candidate`" channels or patches is displayed for your convenience.
+Each CVE contains an identification number, a description of the vulnerability, and links to further information.
+CVE identification numbers use the form ``CVE-YEAR-XXXX``.
 
-You can also assign systems to a [guimenu]``System Set`` for further batch processing.
+In the {productname} {webui}, navigate to menu:Audit[CVE Audit] to see a list of all clients and their current patch status.
+
+By default, the CVE data is updated at 2300 every day.
+We recommend that before you begin a CVE audit you refresh the data to ensure you have the latest patches.
 
 
 
-[[ref-api-usage]]
-== API Usage
+.Procedure: Updating CVE Data
+. In the {productname} {webui}, navigate to menu:Admin[Task Schedules] and select the ``cve-server-channels-default`` schedule.
+. Click btn:[cve-server-channels-bunch].
+. Click btn:[Single Run Schedule] to schedule the task.
+Allow the task to complete before continuing with the CVE audit.
 
-An API method called `audit.listSystemsByPatchStatus` is available to run CVE audits from custom scripts.
-Details on how to use it are available in the API guide.
 
 
+.Procedure: Verifying Patch Status
+. In the {productname} {webui}, navigate to menu:Audit[CVE Audit].
+. OPTIONAL: To check the patch status for a particular CVE, type the CVE identifier in the [guimenu]``CVE Number`` field.
+. Select the patch statuses you want to look for, or leave all statuses checked to look for all.
```

Comment:
ATM, I'm not sure about (US) English plural usuage of "status".  Traditionally, it is also "status" (with a long u, though).  Maybe, we can use "patch state" and "patch states".

Reply:
My AHD (en-US) and my Macquarie (en-AU) don't give a plural. My Writers & Editors dictionary (en-GB) says the plural is `statuses`.

Reply:
Googling brought me here, which seems to agree that `statuses` is common en-US usage over `status`: https://en.wikipedia.org/wiki/Talk%3AStatus